src/libs/ReactNativeLauncher.js

```diff
@@ -205,14 +206,32 @@ class ReactNativeLauncher extends Launcher {
   }
 
   async _start({ initKonnectorError } = {}) {
-    const { account } = this.getStartContext()
+    const { account, konnector } = this.getStartContext()
     try {
       if (initKonnectorError) {
         log.info('Got initKonnectorError ' + initKonnectorError.message)
         throw initKonnectorError
       }
       await this.pilot.call('setContentScriptType', 'pilot')
       await this.worker.call('setContentScriptType', 'worker')
+      const shouldLogout = await this.cleanCredentialsAccounts(konnector.slug)
+      if (shouldLogout) {
+        log(
+          'info',
+          `Detected removed account: first ensure webview is not authenticated`
+        )
+        const cliskVersion = await this.pilot.call('getCliskVersion')
+        if (semverCompare(cliskVersion, '0.10.0') === -1) {
+          log(
+            'warn',
+            'The cozy-clisk version of this konnector is too low: ' +
+              cliskVersion +
+              ' 0.10.0 version should be used to be able to call ensureNotAuthenticated'
```

Comment:
Il manque une ponctuation je pense entre les deux numéros de version.

Ca va sortir une erreur du genre `[...]is too low: 0.9.0 0.10.0 version should be [...]`